From df76d5893b225ffa4c85e7f33fac0c1510b73725 Mon Sep 17 00:00:00 2001
Message-Id: <df76d5893b225ffa4c85e7f33fac0c1510b73725.1421709247.git.chang-joon.lee@intel.com>
In-Reply-To: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
References: <3650eb44734ad4fbe098313349da8185595a20d8.1421709247.git.chang-joon.lee@intel.com>
From: Gajanan Bhat <gajanan.bhat@intel.com>
Date: Thu, 9 Oct 2014 17:10:38 +0530
Subject: [PATCH 54/75] MUST_REBASE [VPG]: adf: Changed ADF header definitions
 as per AOSP changes

Fixes 64 bit related alignment issues.

MUST_REBASE: Since Google ADF framework is not in upstream and also
we cannot have two display drivers, other one being i915, we need to
work on i915 and ADF convergence path before it can be upstreamed

For: GMINL-3547
Change-Id: I982e3971cf56f514ce1f2a9677816d2accf9e49b
Signed-off-by: Peter Collins <peter.collins@intel.com>
Signed-off-by: Gajanan Bhat <gajanan.bhat@intel.com>
Signed-off-by: Uma Shankar <uma.shankar@intel.com>
Signed-off-by: Shobhit Kumar <shobhit.kumar@intel.com>
---
 drivers/video/adf/adf_fops.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- linux.orig/drivers/video/adf/adf_fops.c
+++ linux/drivers/video/adf/adf_fops.c
@@ -187,7 +187,7 @@ static int adf_buffer_import(struct adf_
 		buf->dma_bufs[i] = dma_buf_get(user_buf.fd[i]);
 		if (IS_ERR(buf->dma_bufs[i])) {
 			ret = PTR_ERR(buf->dma_bufs[i]);
-			dev_err(&dev->base.dev, "importing dma_buf fd %d failed: %d\n",
+			dev_err(&dev->base.dev, "importing dma_buf fd %u failed: %d\n",
 					user_buf.fd[i], ret);
 			buf->dma_bufs[i] = NULL;
 			goto done;
